packages/sanity/src/core/form/inputs/PortableText/object/modals/PopoverModal.tsx

```diff
@@ -24,18 +25,8 @@ interface PopoverEditDialogProps {
 
 const POPOVER_FALLBACK_PLACEMENTS: PopoverProps['fallbackPlacements'] = ['top', 'bottom']
 
-export function PopoverEditDialog(props: PopoverEditDialogProps) {
+export function PopoverEditDialog(props: PopoverEditDialogProps): ReactNode {
   const {floatingBoundary, referenceBoundary, referenceElement, width = 1} = props
-  const [open, setOpen] = useState(false)
-
-  // This hook is here to set open after the initial render.
-  // If rendered immediately, the popover will for a split second be
-  // visible in the top left of the boundaryElement before correctly
-  // placed pointing at the reference element.
-  useEffect(() => {
-    setOpen(true)
-  }, [])
-
   return (
```

Comment:
This is no longer needed after https://github.com/sanity-io/ui/commit/c95c1c014a311747286491781db0bd630f43d63b

Reply:
Nice!